src/lib/progress-spinner/progress-spinner.html

```diff
@@ -4,8 +4,21 @@
   element containing the SVG. `focusable="false"` prevents IE from allowing the user to
   tab into the SVG element.
 -->
-<svg viewBox="0 0 100 100"
-     preserveAspectRatio="xMidYMid meet"
-     focusable="false">
-  <path #path [style.strokeWidth]="strokeWidth"></path>
+
+<svg
+  [style.width.px]="_elementSize"
+  [style.height.px]="_elementSize"
+  [attr.viewBox]="_viewBox"
+  preserveAspectRatio="xMidYMid meet"
+  focusable="false">
+
+  <circle
+    cx="50%"
+    cy="50%"
+    [attr.r]="_circleRadius"
+    [style.animation-name]="'mat-progress-spinner-stroke-rotate-' + diameter"
+    [style.stroke-dashoffset.px]="_strokeDashOffset"
+    [style.stroke-dasharray.px]="_strokeCircumference"
+    [style.transform.rotate]="'360deg'"
```

Comment:
Why do we need this? It doesn't look like it does anything.

Reply:
You know, I thought that was from your commit, but now I have no idea. Maybe it's a weird rebase error? Don't remember adding it.

Reply:
Hmm, I haven't pulled in your changes into my local branch yet. I tried checking it out and it's not in there.

Reply:
I must have been testing something at some point? I'll remove it, in any case.